operator/roles/forkliftcontroller/templates/api/mutatingwebhookconfiguration-forklift-api.yml.j2

```diff
@@ -40,4 +40,28 @@ webhooks:
     scope: Namespaced
   sideEffects: None
   timeoutSeconds: 30
-
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: {{ api_service_name }}
+      namespace: {{ app_namespace }}
+      path: /plan-mutate
+      port: 443
+  failurePolicy: Fail
+  matchPolicy: Equivalent
+  name: plans.forklift.konveyor
+  namespaceSelector: {}
+  objectSelector: {}
+  rules:
+  - apiGroups:
+    - forklift.konveyor.io
+    resources:
+    - plans
+    apiVersions:
+    - v1beta1
+    operations:
+    - CREATE
+    - UPDATE
```

Comment:
how about having it only with create?

Reply:
@liranr23 the logic in this mutator should also apply to 'update', could you elaborate on why not to handle the case of 'update' here?

Reply:
true it make sense also on update, but that would be on the user fault? i am thinking about future handling of new plans with annotation of labels.

Reply:
I don't think we can tell users that drop a transfer network of a plan that they do something invalid
we can still have code in this mutator that applies only to 'create' by checking `ar.Request.Operation == admissionv1.Create`